bip-musig2.mediawiki

```diff
@@ -430,17 +430,17 @@ Input:
 * Run ''PartialSigVerifyInternal(psig, pubnonce<sub>i</sub>, pk<sub>i</sub>, session_ctx)''
 * Return success iff no failure occurred before reaching this point.
 
-'''''PartialSigVerifyInternal(psig, pubnonce, pk<sup>*</sup>, session_ctx)''''':
+'''''PartialSigVerifyInternal(psig, pubnonce, pk, session_ctx)''''':
```

Comment:
There are still occurences of `pk<sup>*</sup>` in the remarks section.

Reply:
fixed